msu/hooks/states/world_state.nut

```diff
@@ -300,6 +300,10 @@
 	{
 		::MSU.System.ModSettings.flagSerialize();
 		::World.Flags.set("MSU.LastDayMorningEventCalled", ::World.Assets.getLastDayMorningEventCalled());
+		foreach(func in ::MSU.System.Serialization.SerializeFunctions)
+		{
+			func(_out);
+		}
```

Comment:
so this is pedantic but I think we've been pretty good (just checked and not perfect but pretty good) about not using shortened words instead of full ones as it's not necessary. I'd just use function instead of here and the same for below

Reply:
Sure, is done